add a list of delicious teas

coleww opened the pull request:
culled from wikipedia :tea:

hugovk commented on data/foods/tea.json:
```diff
@@ -0,0 +1,185 @@
+    "Noon Chai",
+    "Oksusucha",
+    "Oolong",
+    "Oolong",
```
Duplicate Oolongs.

hugovk commented on data/foods/tea.json:
```diff
@@ -0,0 +1,185 @@
+    "Tieguanyin",
+    "Tieluohan",
+    "Tsheringma",
+    "Tulsi or Holy Basil",
```
Perhaps split into two entries?

hugovk commented on data/foods/tea.json:
```diff
@@ -0,0 +1,185 @@
+    "Peace Iced",
+    "Peace",
+    "Peppermint",
+    "Peppermint",
```
Duplicate.

hugovk commented on data/foods/tea.json:
```diff
@@ -0,0 +1,185 @@
+    "Peppermint",
+    "Pouchong",
+    "Pu-erh",
+    "Pu-erh",
```
Duplicate.

hugovk commented:
Generally I'd encourage things to be in sentence case, that is: only cap things like proper nouns, so they can be slotted in anywhere in a sentence without looking odd. For example:

> Would you like a nice hot cup of `peppermint` tea? Or perhaps `Thai` tea?

Instead of:

> Would you like a nice hot cup of `Peppermint` tea? Or perhaps `Thai` tea?

If I want them all lowercase, or want them in Initial Case, or ALL CAPS, or want to cap the first word at the start of a sentence, that's easy with post-processing. But trying to guess correct caps after the fact is impossible.

coleww commented:
@hugovk Sorry about all the dupes, thought I had cleaned it up better :dash: 

I went through and fixed the casing as best i could. Learned a lot about teas from googling every word on the list that I thought might be a proper/place-noun.

I'll keep that in mind next time i make a list from wiki data that I shouldn't just accept their capitalized article titles :<
